sphinx_gallery/gen_gallery.py

```diff
@@ -14,11 +14,13 @@
 import copy
 import re
 import os
+import os.path as op
```

Comment:
I know this is a common convention for some. We have seen it inside some PR for sphinx-gallery. Nevertheless we prefer to be explicit and write `os.path` every time.